kentikapi/client_integration_test.go

```diff
@@ -5,32 +5,39 @@ import (
 	"encoding/json"
 	"fmt"
 	"io/ioutil"
+	"net"
 	"net/http"
 	"net/http/httptest"
 	"net/url"
 	"testing"
 	"time"
 
+	syntheticspb "github.com/kentik/api-schema-public/gen/go/kentik/synthetics/v202101beta1"
 	"github.com/kentik/community_sdk_golang/kentikapi"
-	"github.com/kentik/community_sdk_golang/kentikapi/synthetics"
+	httpSynthetics "github.com/kentik/community_sdk_golang/kentikapi/synthetics"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/status"
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/timestamppb"
 )
 
 const (
 	testAgentID = "968"
 )
 
-func TestClient_PatchAgent(t *testing.T) {
+func TestClient_PatchAgentHTTP(t *testing.T) {
```

Comment:
I strongly prefer that the integration test uses only gRPC. I may be incomplete/failing for some time, but eventually it should work exactly as it's HTTP/REST counterpart. Ideally, we should have test data (stored in JSON or some other suitable format) that are used both by the HTTP/REST and gRPC fake servers, so that both HTTP and gRPC clients using them are guaranteed to get the same responses for equivalent requests. Do you see any problem with this approach?